workspaces/util/src/__tests__/timestampToYearMonthDay-test.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { timestampToYearMonthDay } from "../timestampToYearMonthDay";
+
+describe("timestampToYearMonthDay", () => {
+  it("returns formated timestamp correctly with default separator", () => {
+    expect(timestampToYearMonthDay(1672531199)).toBe("2022-12-31"); // Saturday, December 31, 2022
+  });
+
+  it('returns formated timespamp with custom seperator "/"', () => {
+    expect(timestampToYearMonthDay(1672531199, "/")).toBe("2022/12/31"); // Saturday, December 31, 2022
+  });
+  
+  it("handle day different timestamps in different years ", () => {
```

Comment:
I think this description needs a bit more clarity
```suggestion
  it("handles timestamps across different years", () => {
```